SuperBuild/External_SimpleITK.cmake

```
@@ -161,6 +161,12 @@ ExternalProject_Execute(${proj} \"install\" \"${PYTHON_EXECUTABLE}\" \"-m\" \"pi
     NAME ${proj}
     )
 
+  if(USE_GOLD_LINKER AND _COKMPILER_SUPPORTS_GOLD_LINKER)
+    set(SimpleITK_USES_GOLD_LINKER ON)
+  else()
+    set(SimpleITK_USES_GOLD_LINKER OFF)
+  endif()
+
```

```suggestion
  if(Slicer_COMPILER_SUPPORTS_GOLD_LINKER)
    list(APPEND EXTERNAL_PROJECT_OPTIONAL_CMAKE_CACHE_ARGS
      -DUSE_GOLD_LINKER:BOOL=${Slicer_USE_GOLD_LINKER}
      )
  endif()
```

and move the block above where `EXTERNAL_PROJECT_OPTIONAL_CMAKE_CACHE_ARGS` is updated.

(Note there was also a type ... `_COKMPILER_SUPPORTS_GOLD_LINKER` -> `_COMPILER_SUPPORTS_GOLD_LINKER`)